src/main-app.component.ts

```diff
@@ -1,14 +1,16 @@
-import { Component } from '@angular/core';
+import {Component, ViewEncapsulation} from '@angular/core';
 import { ROUTER_DIRECTIVES } from '@angular/router';
 
 import { SideMenuComponent } from './components/sidemenu/side-menu.component';
+import { NotificationComponent } from './components/notification/notification.component';
 
 @Component({
     moduleId: module.id,
     selector: 'main-app',
     template: require('./main-app.component.html'),
     styles: [require('./main-app.component.scss')],
-    directives: [SideMenuComponent, ROUTER_DIRECTIVES]
+    directives: [SideMenuComponent, ROUTER_DIRECTIVES, NotificationComponent],
+    encapsulation: ViewEncapsulation.None
```

Comment:
This is our main entry point into our application. So I changed this to none, so we can have some global CSS styles like select2 style. Select2 drop down is added dynamically before the end of the body tag. So it's outside out main-app tag and we can't style it. I would use main-app.component.scss only for this type of styling, everything else is inside correct component.